src/graphql/mutations/events.ts

```diff
@@ -19,6 +19,32 @@ export const ADD_EVENT = gql`
   }
 `;
 
+export const EDIT_EVENT = gql`
+  mutation updateEvent(
+    $id: ID!
+    $name: String!
+    $eventDate: Date!
+    $createdBy: ID!
+    $isActive: Boolean!
+  ) {
+    updateEvent(
+      id: $id
+      name: $name
+      eventDate: $eventDate
+      createdBy: $createdBy
+      isActive: $isActive
+    ) {
+      name
```

Comment:
You have to return the event `id` here for the cache to update automatically. Then you probably won't need to call `refetch()` like you're doing :)

Reply:
I added the event id here, but the cache isn't updating automatically without `refetch()`. I need to refresh my browser for the updates to show.